openpdf/src/main/java/com/lowagie/text/pdf/PdfDocument.java

```diff
@@ -1920,7 +1920,7 @@ void calculateOutlineCount() {
      * Recursive method to update the count in the outlines.
      */
     void traverseOutlineCount(PdfOutline outline) {
-        ArrayList kids = outline.getKids();
+        java.util.List<PdfOutline> kids = outline.getKids();
```

Comment:
Seems like a `import java.util.List` is missing.

Reply:
Not really. There is already an import for com.lowagie.text.List there, so we need here the FQN.